packages/babel-plugin-ember-test-metadata/__tests__/utils-test.js

```diff
@@ -63,80 +59,17 @@ describe('getNormalizedFilePath', () => {
         'acceptance',
         'my-test.js'
       ),
-    },
-    normalFilePath: {
-      root: '',
-      filename: path.join('this', 'is', 'not-an-embroider', 'path'),
-    },
-  };
-  const projectConfiguration = {
-    pkg: {
-      name: 'test-app',
-      'ember-addon': {},
-    },
-  };
-
-  it('returns stripped file path as expected', () => {
-    expect(getNormalizedFilePath(fileOpts.embroiderBuildPath, projectConfiguration)).toBe(
+    }, projectConfiguration)).toBe(
```

Comment:
Let's avoid using `toBe` here, as it's not intuitive what you're expecting to assert here. I know we have string -> string comparison, but in that case I think `toEqual` makes much more sense (and reads better).

Reference: https://dev.to/thejaredwilcurt/why-you-should-never-use-tobe-in-jest-48ca